### DISCUSSION TITLE
NodOn Soft Button -  D2-03-0A

### DISCUSSION THREAD

**Discussion by SmartHomeForDummies** [mak-gitdev/HA_enoceanmqtt#85] [Request]
Friendly request to integrate the[ NodOn Soft Button](https://nodon.fr/wp-content/uploads/2018/12/NodOn_TheSoftButton_EnOcean_Datasheet_EN_181210.pdf)
EEP D2-03-0A

I create the device configuration like this.

[NodOn_softButton]
address         = 0xXXXXXXXX
rorg            = 0xD2
func            = 0x03
type            = 0x0A

But unfortunately it doesn't work.
I assume that the EEP ist not supported yet?
![NodOn Soft Button](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/13818392/921690a1-7574-4641-8926-a7768d4d2b07)

**Comment by SmartHomeForDummies** [comment]
Now this is a little embarrassing!
The battery was empty! 🫣

**Comment by SmartHomeForDummies** [comment]
Can be closed.
![NodOn Soft Button - Does what he's supposed to](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/13818392/fd913a93-df12-4a7b-ad27-4d9137a7b8cb)

**Comment by mak-gitdev** [comment]
Hi @SmartHomeForDummies,

Please have a look at this [wiki](https://github.com/mak-gitdev/HA_enoceanmqtt/wiki/Using-the-NodOn-soft-button-%28D2%E2%80%9003%E2%80%900A%29). It might be very helpful when using D2-03-0A.
